services/crates/crates-downloads.service.js

```diff
@@ -59,7 +81,25 @@ module.exports = class CratesDownloads extends BaseCratesService {
     }
   }
 
+  _extractDownloads({ variant, json }) {
```

Comment:
Our convention is to use the name `transform` for this type of function that takes the raw response and returns the data points, so let's rename this
```suggestion
  transform({ variant, json }) {
```

Reply:
Done!